src/transport_picoquic.cpp

```diff
@@ -739,6 +740,27 @@ PicoQuicTransport::closeStream(const TransportContextId& context_id, const Strea
     deleteStreamContext(context_id, stream_id);
 }
 
+bool PicoQuicTransport::getPeerAddrInfo(const TransportContextId& context_id,
+                                        sockaddr_storage* addr)
+{
+    // Locate the specified transport context
```

Comment:
Not here, but we need to add locking to this post merge and merge of the streaming changes that adds the lock to use.

Reply:
Locking is needed in several places, as maps are consulted via several API calls.